report/report.py

```diff
@@ -100,3 +112,14 @@ def make_report_embed(self, ctx: commands.Context, message: str):
             name="Message", value=escape(message or "<no message>"), inline=False
         )
         return data
+
+    def make_reporter_reply(self, ctx: commands.Context, message: str, emergency: bool) -> discord.Embed:
+        report_reply_embed = discord.Embed(
+            title="Report Received", colour=discord.Color.orange() if emergency else ctx.guild.me.colour
+        )
+        report_reply_embed.add_field(name="Server", value=ctx.guild.name)
```

Comment:
It'd be nice to have these fields be inline, to fit the embed that is sent to modchat

Reply:
Made the embed nearly identical to the mod alert.